winrt-related-src/toolkits/winui3/index.md

```diff
@@ -148,13 +148,15 @@ The following list shows the UI controls and features that are not included (or
 * [MediaPlayerElement](https://docs.microsoft.com/uwp/api/windows.ui.xaml.controls.mediaplayerelement), [MediaElement](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.MediaElement) and related APIs
 * [RevealBrush](https://docs.microsoft.com/uwp/api/windows.ui.xaml.media.revealbrush)
 * [SwapChainPanel](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.SwapChainPanel) and [SwapChainBackgroundPanel](https://docs.microsoft.com/uwp/api/windows.ui.xaml.controls.swapchainbackgroundpanel)
-* [System.ComponentModel.INotifyPropertyChanged](https://docs.microsoft.com/dotnet/api/system.componentmodel.inotifypropertychanged?view=dotnet-uwp-10.0) and [System.Collections.Specialized.INotifyCollectionChanged](https://docs.microsoft.com/dotnet/api/system.collections.specialized.inotifycollectionchanged?view=dotnet-uwp-10.0) for C# apps. Instead, use the ```Microsoft.UI.Xaml.Interop``` versions of these interfaces
```

Comment:
To keep the list complete, I think we should leave this text to here but add a "see below for more info" link.